### ISSUE TITLE
Method annotated as @Step with null parameter values throws an error

### ISSUE THREAD

**Issue by ASaunin** [allure-framework/allure2#350]
Code example (message!=null, actual==null, expected==null):
```
@Step("Text \"{actual}\" is equal to expected \"{expected}\"")
private static void checkText(Sms message, String actual, String expected) {
    assertThat(actual)
            .as("Check %s body", message)
            .isEqualTo(expected);
}
```
Stack trace:
```
java.lang.IllegalStateException: Can not cast io.qameta.allure.model.TestResult@3adb4cc8 to class io.qameta.allure.model.StepResult

	at io.qameta.allure.internal.AllureStorage.cast(AllureStorage.java:131)
	at io.qameta.allure.internal.AllureStorage.get(AllureStorage.java:115)
	at io.qameta.allure.internal.AllureStorage.getStep(AllureStorage.java:91)
	at io.qameta.allure.AllureLifecycle.updateStep(AllureLifecycle.java:209)
	at io.qameta.allure.AllureLifecycle.updateStep(AllureLifecycle.java:205)
	at io.qameta.allure.aspects.StepsAspects.stepFailed(StepsAspects.java:67)
	at com.asaunin.AbstractDeliveryTest.checkDeliveryBySmsc(AbstractDeliveryTest.java:118)
	at com.asaunin.AbstractDeliveryTest.checkDelivery(AbstractDeliveryTest.java:48)
	at com.asaunin.BasicSynchronousDeliveryTest.shouldDeliverMessage(BasicSynchronousDeliveryTest.java:49)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.MethodInvocationHelper$1.runTestMethod(MethodInvocationHelper.java:209)
	at org.springframework.test.context.testng.AbstractTestNGSpringContextTests.run(AbstractTestNGSpringContextTests.java:175)
	at org.testng.internal.MethodInvocationHelper.invokeHookable(MethodInvocationHelper.java:221)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:657)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:72)
	at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:127)
```
Passing non-null values as a method parameters, doesn't throw any exceptions

**Comment by sskorol** [comment]
@ASaunin thanks, just raised PR with a fix.

**Comment by ASaunin** [comment]
Sry, I'll keep it in touch in the future.

**Comment by baev** [comment]
fixed via https://github.com/allure-framework/allure-java/commit/3b4d32d20242e7d961f5d2121ae52621d497a73d

**Comment by baev** [comment]
fix released in 2.0-BETA7

**Comment by ASaunin** [comment]
@baev Thx
